### ISSUE TITLE
Saving Training model and then running it on test dataset

### ISSUE THREAD

**Issue by spartian** [AlexGidiotis/Document-Classifier-LSTM#6]
Hello,
As I understand, there has to be(very often), training set,validation set, test set. Machine learning model trains on training set and the parameters are tuned at validation set. This model is then saved and test set is then applied at that model. However going through, hatt.py, I saw the model being saved as checkpoint. But I did not find any implementation of using that model for test set. Am I right or have i missed something in the code? The model should be saved and then applied to test set. In your case, did you not use the testing set?

**Comment by AlexGidiotis** [comment]
You haven't missed something. It looks like I have not uploaded the eval code (probably I forgot to). Although, it should be straightforward to run on a test set.

**Comment by spartian** [comment]
Could you please upload it? I am not proficient in keras and python. I am looking at your code and trying to understand it. I would be very grateful. Thank you

**Comment by spartian** [comment]
Another thing...Have you implemented multi layer perceptron with a single layer? The original paper says it uses MLP to get hidden representation of hit i.e. uit (annotation of words)

**Comment by spartian** [comment]
Another thing...Have you implemented multi layer perceptron with a single layer? The original paper says it uses MLP to get hidden representation of hit i.e. uit (annotation of words)

**Comment by AlexGidiotis** [comment]
This is implemented in the attention layer for word and sentence attentions.

**Comment by spartian** [comment]
Ok.....I think I missed the detail in attention context ....will definitely look into it.
